docroot/modules/custom/va_gov_build_trigger/src/Service/BuildFrontend.php

```diff
@@ -178,10 +178,11 @@ public function triggerFrontendBuildFromContentSave(NodeInterface $node) {
         return;
       }
       $this->triggerFrontendBuild();
+      // We previously also showed this message to the user, but the build
+      // trigger mechanism also needs to show a message, so this was redundant.
+      // We still send logs as they may be useful for diagnostics.
```

Comment:
minor:  I'd prefer  to let the commits tell the story of history of anything no longer in code.   Leaving a bunch of grave markers in the code is going to get messy and wordy over time.

Reply:
The commits don't tell the story because we squash and all useful intermediary commit messages are lost. We'll have no history of why this was removed. I think squashing is good, mind you, but it makes code spelunking more difficult. 

This comment is there to explain the contrast to the if condition just above this, where we *do* show a message. If I were looking at this naively, I'd wonder why we set a message when a build isn't triggered, but we don't set one when we do trigger.

Reply:
They do still tell the story because we squash and merge, not fixup and merge  No merge commit comments long or short are lost in the process.
![image](https://user-images.githubusercontent.com/5752113/155405758-4718e9f7-476a-4e1b-a6b0-827d7629ecb7.png)

Reply:
```suggestion
      // Since a build requested message is already displayed, just log what triggered it.
```
This explains why, without tales of yesteryear ;)